src/internal/util/NodeEventEmitterDataType.ts

```diff
@@ -0,0 +1,236 @@
+/**
+ * Issue #6669 - fix(fromEvent): infer from Node.js EventEmitter with types
```

Comment:
I'm not sure how these comments will affect our documentation system. Probably should just stick regular doc comments as seen elsewhere in the code base.

Reply:
I have removed them.